### PR TITLE
rename space manager to space admin

### PR DESCRIPTION
**Pull request by wkloucek**
## Description
The global "Space Manager" role is now called "Space Admin" to be different from the per space "Space Manager"

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance test [<strong>Core-API-Tests-ocis-storage-4</strong>](https://drone.owncloud.com/owncloud/ocis/10012/26/1) failed. Further test are cancelled...

File: docs/ocis/getting-started/demo-users.md [hunk 1]
```diff
@@ -16,13 +16,13 @@ To skip the generation of demo users, run the inital setup step with an addition
...
+| username  | password      | email                 | role        | groups                                                                  |
+| --------- | ------------- | --------------------- | ----------- | ----------------------------------------------------------------------- |
+| admin     | admin         | admin@example.org     | admin       | users                                                                   |
+| einstein  | relativity    | einstein@example.org  | user        | users, philosophy-haters, physics-lovers, sailing-lovers, violin-haters |
+| marie     | radioactivity | marie@example.org     | user        | users, physics-lovers, polonium-lovers, radium-lovers                   |
+| moss      | vista         | moss@example.org      | admin       | users                                                                   |
+| richard   | superfluidity | richard@example.org   | user        | users, philosophy-haters, physics-lovers, quantum-lovers                |
+| katherine | gemini        | katherine@example.org | space admin | users, sailing-lovers, physics-lovers, quantum-lovers                   |
```
Comment (kobergj) [inline docs/ocis/getting-started/demo-users.md:26]: She is not a `user` any more? She used to be before? Why is this happening?

**Review by kobergj — CHANGES_REQUESTED** [review]

File: docs/ocis/getting-started/demo-users.md (hunk 1 above)
Comment (wkloucek) [reply docs/ocis/getting-started/demo-users.md:26]: users cannot be part of two roles... so admin incorporates users permissions and so needs a space manager (see https://github.com/owncloud/ocis/issues/3302)

Comment (kobergj) [reply docs/ocis/getting-started/demo-users.md:26]: Ok. Commented on the ticket. Thanks!

**Review by kobergj — APPROVED** [review]
Sonarcloud doesn't seem to like you :wink:  But looks good to me...

**Pull request merged**